src/test/java/zippopotamus/steps/PostalCodeSteps.java

```diff
@@ -0,0 +1,41 @@
+package zippopotamus.steps;
+
+
+import io.cucumber.java8.En;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.junit.Assert;
+
+public class PostalCodeSteps implements En {
+
+    private Response response;
+    private String getLocationInformationByCountryAndPostalcode = "http://zippopotam.us/country/postalCode";
+    private String getLocationInformationByCountry_StateAndCity = "http://zippopotam.us/country/state/city";
```

Comment:
Since both urls contain the base string 'http://zippopotam.us', we could store that in another variable, and append it to the rest of the string for each variable, something like this: 
`private static final String BASE_ZIPPOPOTAMUS_URL =  "http://zippopotam.us/"; 
rivate String getLocationInformationByCountryAndPostalcode = BASE_ZIPPOPOTAMUS_URL  + "country/postalCode";`

Reply:
Done